region/lowernorfair/east/Lower Norfair Escape Power Bomb Room.json

```diff
@@ -93,6 +94,32 @@
         "Holding down and mashing forward sometimes works, or X-Ray can be used at the end of the Crystal Flash to more easily time the inputs."
       ]
     },
+    {
+      "link": [1, 3],
+      "name": "Lower Norfair Jail Grapple Teleport",
+      "notable": true,
+      "entranceCondition": {
+        "comeInWithGrappleTeleport": {
+          "blockPositions": [[12, 13]]
+        }
+      },
+      "requires": [
+        "Morph",
+        {"heatFrames": 180}
+      ],
+      "note": [
+        "In the previous room, avoid triggering the transition too deeply, otherwise Samus can end up stuck too far right in the wall.",
+        "After teleporting, Samus should be standing inside the wall.",
+        "Wait a moment for the acid to lower, then retract Grapple by pressing up, which will pull Samus left and down.",
+        "Hold left, and release Grapple while still holding left.",
+        "Perform a turn-around spin jump (to the right).",
+        "Then morph and roll out to the left."
+      ],
+      "devNote": [
+        "Coming in from position [12, 12] can also work but is more difficult:",
```

Comment:
Looks like my bottom position save was too deep but my top position save had no problem with this.

Reply:
It works if I hold right while hitting the transition, as opposed to coming to a stop inside the transition and then pressing right. 👍